src/main/java/com/kodilla/ecommercee/domain/ProductDto.java

```diff
@@ -1,4 +1,19 @@
 package com.kodilla.ecommercee.domain;
 
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
 public class ProductDto {
+    private long id;
+    private String name;
+    private String description;
+    private BigDecimal price;
+    private boolean availability;
+    private ProductGroup productGroup;
```

Comment:
Jeśli mamy klasę DTO, to nie powinna ona mieć relacji do encji (ProductGroup). Wystarczy tam albo same id grupy (Long groupId) albo jeśli chcesz podawać cały obiekt to powinien tam być ProductGroupDto